crates/matrix-sdk-base/src/store/traits.rs

```diff
@@ -1156,7 +1157,16 @@ pub struct QueuedEvent {
     /// If the event couldn't be sent because of an API error, it's marked as
     /// wedged, and won't ever be peeked for sending. The only option is to
     /// remove it.
-    pub is_wedged: bool,
+    pub error: Option<QueueWedgeError>,
```

Comment:
Should we update this doc comment, since an identical one was added below to `is_wedged()`?